webdriver/tests/bidi/storage/set_cookie/partition.py

```diff
@@ -1,34 +1,46 @@
 import pytest
 from webdriver.bidi.modules.storage import BrowsingContextPartitionDescriptor, StorageKeyPartitionDescriptor
 from .. import assert_cookie_is_set, create_cookie
+from ... import recursive_compare
 
 pytestmark = pytest.mark.asyncio
 
 
-async def test_partition_context(bidi_session, set_cookie, top_context, test_page, origin, domain_value):
+def assert_set_cookie_result(set_cookie_result, partition):
```

Comment:
should it be moved to the init file?

Reply:
It's not used anywhere else, so no reason for now